spec/queries/drug_stocks_query_spec.rb

```diff
@@ -0,0 +1,140 @@
+require "rails_helper"
+
+RSpec.describe DrugStocksQuery do
+  let!(:state) { "Punjab" }
+  let!(:protocol) { create(:protocol, :with_tracked_drugs) }
+  let!(:facility_group) { create(:facility_group, protocol: protocol) }
+  let!(:user) { create(:admin, :manager, :with_access, resource: facility_group) }
+  let!(:for_end_of_month) { Date.today.end_of_month }
+  let!(:drug_category) { "hypertension_ccb" }
+  let!(:stocks_by_rxnorm) {
+    {"329528" => 10000, "329526" => 20000, "316764" => 10000, "316765" => 20000, "979467" => 10000}
+  }
+  let!(:punjab_drug_stock_config) {
+    {"load_coefficient" => 1,
+     "drug_categories" =>
+        {"hypertension_ccb" => {"new_patient_coefficient" => 1.4, "329528" => 1.2, "329526" => 2},
+         "hypertension_arb" => {"new_patient_coefficient" => 0.37, "316764" => 1, "316765" => 2, "979467" => 1},
+         "hypertension_diuretic" => {"new_patient_coefficient" => 0.06, "316049" => 1, "331132" => 1}}}.with_indifferent_access
```

Comment:
The test data can be simpler since we're mocking this anyway. `1`, `2`, `1.5`, easy coeffs.

Reply:
We picked up these values from one of our state protocols. I think its okay to leave in, will give us realistic tables for sandbox data